drivers/storage/portworx/component/monitoring.go

```diff
@@ -31,14 +39,61 @@ const (
 	PxBackupServiceMonitor = "px-backup"
 	// PxPrometheusRule name of the prometheus rule object for Portworx
 	PxPrometheusRule = "portworx"
+	// GrafanaDeploymentName is the name for the operator managed grafana
+	GrafanaDeploymentName = "grafana"
+	// GrafanaContainerName is the name for the grafana container
+	GrafanaContainerName = "grafana"
 
+	grafanaDefaultCPU    = "0.1"
+	grafanaDefaultMemory = "0.1"
+	grafanaPort          = 3000
+	grafanaLoginEndpoint = "/login"
 	pxPrometheusRuleFile = "portworx-prometheus-rule.yaml"
 )
 
+var (
+	grafanaDeploymentVolumes = []corev1.VolumeSpec{
+		{
+			Name:      "grafana-source-config",
+			MountPath: "/etc/grafana/provisioning/datasources",
+			VolumeSource: v1.VolumeSource{
+				ConfigMap: &v1.ConfigMapVolumeSource{
+					LocalObjectReference: v1.LocalObjectReference{
+						Name: "grafana-dashboards",
```

Comment:
We will have to create these config maps, right?

Reply:
Yes creating them now in grafana.go